test/common/testlib.py

```diff
@@ -2059,7 +2139,7 @@ def snapshot(self, title: str, label: str | None = None) -> None:
         if self.browser is not None:
             try:
                 self.browser.snapshot(title, label)
-            except RuntimeError:
+            except Error:
```

Comment:
aside: this is a pretty bad name for a class... of course, this is not your fault..